### PR TITLE
cpu: doxygen cleanup

### PR DESCRIPTION
**Pull request by aabadie**
Another pass of cleanup of doxygen documentation grouping in the cpu modules:
* remaining issue with MIPS
* msp430

### CODE REVIEW

File: cpu/mips32r2_common/include/cpu.h [hunk 1]
```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @defgroup    cpu_mips32r2_commom MIPS32R2 Common
+ * @defgroup    cpu_mips32r2_commom Microchip MIPS32R2 Common
```
Comment (miri64) [inline cpu/mips32r2_common/include/cpu.h]: Isn't MIPS by Imagination?

Comment (aabadie) [inline cpu/mips32r2_common/include/cpu.h]: oh, yes maybe...

Comment (aabadie) [inline cpu/mips32r2_common/include/cpu.h]: fixed

**Comment by aabadie** [comment]
@miri64, should be fixed now.

File: cpu/mips32r2_common/include/cpu_conf.h [hunk 2]
```diff
@@ -6,20 +6,19 @@
...
-
 /**
- * @defgroup    cpu_mips32r2_commom MIPS32R2 Common
- * @ingroup     cpu
+ * @ingroup     cpu_mips32r2_commom
  * @{
  *
  * @file
- * @brief       Common CPU definitions for mip32r2 compatable devices.
+ * @brief       Common CPU definitions for mip32r2 compliant devices.
```
Comment (gebart) [inline cpu/mips32r2_common/include/cpu_conf.h]: mip32r2 -> MIPS32R2

File: cpu/mips_pic32mx/include/cpu.h [hunk 3]
```diff
@@ -9,7 +9,7 @@
  */
 
 /**
- * @defgroup    cpu_mips_pic32mx MIPS PIC32MX
+ * @defgroup    cpu_mips_pic32mx Microchip MIPS PIC32MX
```
Comment (gebart) [inline cpu/mips_pic32mx/include/cpu.h]: ~~MIPS~~: "Microchip PIC32MX"

File: cpu/mips_pic32mz/include/cpu.h [hunk 4]
```diff
@@ -9,7 +9,7 @@
  */
 
 /**
- * @defgroup    cpu_mips_pic32mz MIPS PIC32MZ
+ * @defgroup    cpu_mips_pic32mz Microchip MIPS PIC32MZ
```
Comment (gebart) [inline cpu/mips_pic32mz/include/cpu.h]: "Microchip PIC32MZ"

File: cpu/msp430_common/include/cpu.h [hunk 5]
```diff
@@ -7,20 +7,25 @@
...
  * @ingroup     cpu
  * @brief       Texas Instruments MSP430 specific code
-
-<h2>First steps</h2>
-\li See the <a href="../manual/index.html">manual</a> for toolchain and ide setup
-
+ *
+ * ## First steps
+ *
+ * See the <a href="../manual/index.html">manual</a> for toolchain and ide
```
Comment (gebart) [inline cpu/msp430_common/include/cpu.h]: I have no idea what file this is supposed to point to. It doesn't work on https://api.riot-os.org or https://riot-os.org/api

File: cpu/msp430_common/include/msp430_types.h [hunk 6]
```diff
@@ -89,3 +97,4 @@ typedef     uint32_t useconds_t;  /**< Used for time in microseconds */
 #endif
 
 #endif /* MSP430_TYPES_H */
+/** @} */
```
Comment (gebart) [inline cpu/msp430_common/include/msp430_types.h]: missing newline at EOF

File: cpu/msp430_common/include/cpu.h (hunk 5 above)
Comment (aabadie) [reply cpu/msp430_common/include/cpu.h]: me neither and looking at the [wiki](https://github.com/RIOT-OS/RIOT/wiki/Family:-MSP430) the links are broken.

Comment (aabadie) [reply cpu/msp430_common/include/cpu.h]: What about pointing to the [TI website](http://www.ti.com/tool/msp430-gcc-opensource#Technical%20Documents) ?
I have no experience with MSP430 so maybe others may have a better opinion on that.

**Comment by aabadie** [comment]
@gebart, comments addressed. I'm not what to do with the broken link to the "manual", maybe simply remove it ?

**Comment by gebart** [comment]
@aabadie the toolchain at the link you provided doesn't work yet, it needs newlib support for msp430 in RIOT. Let's just remove the link for now..

**Comment by aabadie** [comment]
> Let's just remove the link for now..

done

**Comment by aabadie** [comment]
@gebart, this one should be fine now.

**Comment by aabadie** [comment]
@gebart, do you have other comments on this one ?

**Pull request merged**